governance-exercises/Polling Exercise #2 Response.md

```diff
@@ -0,0 +1,40 @@
+#### My Interpretation of the MIPs and Plan
+
+[MIP101 2.6.6](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-privacy) does not direct a facilitator to draft a poll in this instance. It states that "If there’s clear evidence or significant suspicion that the identity of an AD has been revealed, a FacilitatorDAO **must immediately derecognize the AD**." then goes on to add "FacilitatorDAOs **must err on the side of caution** and act in case there is **any kind of real possibility that the privacy of an AD is compromised**.". 
+
+The wording could be improved, but in order to "err on the side of caution" an off-boarding by the Facilitator would unfortunately be the appropriate response here. This is consistent with [previous off-boardings](https://forum.makerdao.com/t/governance-scope-actions-pertaining-to-delegates-in-light-of-mip102c2-sp7-approval/20946) of ADs, which were not polled.
```

Comment:
I think this response is fairly justified. I think part of the assumption of the exercise is that it's not clear whether the materials constitute 'significant suspicion' or not. 

The reason previous instances were not polled is because the ADs involved didn't contest the decision, and because the materials were not made public. While I agree there are commonalities, there are also important differences that mean those precedents are not entirely applicable here.